packages/suite/src/views/dashboard/components/T2B1PromoBanner.tsx

```diff
@@ -0,0 +1,268 @@
+        top: 50px;
+    }
+`;
+
+const ProductsImageWrapper = styled.div`
+    grid-column: 2;
+    grid-row: 1 / 3;
+    position: relative;
+    z-index: 0;
+
+    @media screen and (max-width: ${MOBILE_BREAKPOINT}px) {
+        grid-row-end: 4;
+    }
+`;
+
+const ProductsImage = styled(Image)`
+    position: absolute;
+    right: -67px;
+    max-width: none;
+    height: fill-available;
+    filter: none;
+
+    @media screen and (max-width: ${SMALL_SCREEN_BREAKPOINT}px) {
+        right: -57px;
+    }
+`;
+
+const LinkButtonShopNow = styled(TrezorLink)`
+    grid-column: 3;
+    grid-row: 2;
+    height: 42px;
+    margin-bottom: 24px;
+    align-self: end;
+
+    @media screen and (max-width: ${SMALL_SCREEN_BREAKPOINT}px) {
+        margin-bottom: 16px;
+    }
+
+    @media screen and (max-width: ${MOBILE_BREAKPOINT}px) {
+        grid-column: 1;
+        grid-row: 3;
+        width: 128px;
+        height: 36px;
+        margin-bottom: 12px;
+    }
+`;
+
+const ButtonShopNow = styled(Button)`
+    background-color: #9be887;
+    width: 100%;
+    height: 100%;
+
+    color: black;
+
+    :hover {
+        background-color: #7cba6c;
+    }
+`;
+
+const ButtonClose = styled(Button)`
+    grid-column: 4;
+    grid-row: 2;
+    margin-bottom: 24px;
+    align-self: end;
+    z-index: 1;
+
+    /* stylelint-disable color-function-notation */
+    background: ${({ theme }) => rgba(theme.TYPE_WHITE, 0.16)};
+    /* stylelint-enable color-function-notation */
+    height: 42px;
+    width: 42px;
+
+    :hover {
+        background: #0b4936;
+    }
+
+    svg {
+        height: 24px;
+        width: 24px;
+        fill: ${({ theme }) => theme.TYPE_WHITE};
+    }
+
+    @media screen and (max-width: ${SMALL_SCREEN_BREAKPOINT}px) {
+        margin-bottom: 16px;
+    }
+
+    @media screen and (max-width: ${MOBILE_BREAKPOINT}px) {
+        grid-column: 4;
+        grid-row: 1;
+        align-self: auto;
+        margin-bottom: 0;
+        margin-top: 12px;
+        height: 31px;
+        width: 31px;
+
+        svg {
+            height: 17px;
+            width: 17px;
+        }
+    }
+`;
+
+export const T2B1PromoBanner = () => {
+    const dispatch = useDispatch();
+
+    const shouldShowDashboardT2B1PromoBanner = useSelector(selectT2B1PromoBanner);
+    const screenWidth = useSelector(state => state.resize.screenWidth) || 0;
+
+    const onCloseBanner = () => {
+        analytics.report({
+            type: EventType.T2B1DashboardPromo,
+            payload: {
+                action: 'close',
+            },
+        });
+        dispatch(setFlag('showDashboardT2B1PromoBanner', false));
+    };
+
+    const onShopNow = () =>
+        analytics.report({
+            type: EventType.T2B1DashboardPromo,
+            payload: {
+                action: 'shop',
+            },
+        });
+
+    const isMobile = screenWidth <= MOBILE_BREAKPOINT;
+
+    if (!shouldShowDashboardT2B1PromoBanner) return null;
+
+    return (
+        <BannerWrapper>
```

Comment:
Apply [this animation](https://github.com/trezor/trezor-suite/blob/4229fd483bc1dbebc794e60d4875ca26b4bf8512/packages/suite/src/views/settings/SettingsGeneral/DesktopSuiteBanner.tsx#L104) please ;}